pinot-common/src/main/java/com/linkedin/pinot/common/utils/CommonConstants.java

```diff
@@ -332,20 +332,9 @@ public ServerType getServerType() {
     }
   }
 
-  public static class HadoopConstants {
-    public static final String PRINCIPLE = "hadoop.kerberos.principle";
-    public static final String KEYTAB = "hadoop.kerberos.keytab";
-    public static final String HADOOP_CONF_PATH = "hadoop.conf.path";
-
-    public static final String RETRY = "retry.count";
-    public static final int RETRY_DEFAULT = 3;
-    public static final String RETRY_WAITIME_MS = "retry.wait.ms";
-    public static final int RETRY_WAITIME_MS_DEFAULT = 100;
-  }
-
-  public static class SegmentFetcher {
-    public static class HdfsSegmentFetcher {
-      public static final String PRINCIPLE = "hadoop.kerberos.principle";
+  public static class SegmentOperations {
+    public static class HadoopSegmentOperations {
+      public static final String PRINCIPAL = "hadoop.kerberos.principle";
```

Comment:
s/principle/principal/

Reply:
I'm worried uber has already wired this config into their machines; that's why I didn't change that one